package.json

```diff
@@ -48,7 +48,8 @@
     "kcd-scripts": "7.5.1",
     "react": "17.0.1",
     "react-test-renderer": "17.0.1",
-    "typescript": "4.1.2"
+    "typescript": "4.1.2",
+    "eslint": "^7.14.0"
```

Comment:
Renovate used to pin `devDependencies`, but I'm not sure what Dependabot expects since #509.

I don't particularly mind either way, but I'd like them to be consistent. Maybe we unpin them all now?

Reply:
yes right, I have removed the pin :)